mybookmarks@markbokil.com: Change data directory

Centurix opened the pull request:
When updating the applet through the software update, any custom menu settings are lost due to the configuration file being stored in the applets source directory. This commit will move the configuration to the correct users configuration area.

I also note that this applet has no current maintainer, I'd like to volunteer myself as the maintainer for this applet as I also use this applet a lot.

rcalixte commented:
> I also note that this applet has no current maintainer, I'd like to volunteer myself as the maintainer for this applet as I also use this applet a lot.

Feel free to add yourself as the author in the info.json file for this applet.

Centurix commented:
> > I also note that this applet has no current maintainer, I'd like to volunteer myself as the maintainer for this applet as I also use this applet a lot.
> 
> Feel free to add yourself as the author in the info.json file for this applet. This also looks like it might need a rebase as the previous changes are still showing as a diff.

Rebased and changed the author. Thanks!

rcalixte commented:
> Rebased and changed the author. Thanks!

Thank you for taking this on!
